include/sslopt-vars.h

```diff
@@ -85,6 +85,7 @@ ulong opt_ssl_fips_mode = SSL_FIPS_MODE_OFF;
 bool ssl_mode_set_explicitly = false;
 char *opt_ssl_session_data = nullptr;
 bool opt_ssl_session_data_continue_on_failed_reuse = false;
+char *opt_tls_sni_servername = nullptr;
```

Comment:
is this variable introduced by us or by server?

Reply:
upstream, this part of the code in the else branch of a preprocessor macro `#ifndef XTRABACKUP` upstream change got applied there (line 70) but the else branch needed this duplication.